.github/env/_build_image.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+export VLLM_VER=v0.6.1
```

Comment:
Current Dockerfiles use `v0.8.3` for the CPU version, only OpenVINO Dockerfile uses the ancient `v0.6.1` vLLM.

=> Either add 3rd variable for OpenVINO version, or use same version for it as for CPU.

Reply:
It would be good to update OpenVINO support, first to something closer to v0.8.1, and eventually to newer vLLM version using the new vLLM OpenVINO plugin project: https://github.com/vllm-project/vllm-openvino

But that's definitely something out of scope for this PR.